.github/workflows/pr.yml

```
@@ -27,11 +24,14 @@ jobs:
   QC:
     needs: [Checkout]
     uses: "./.github/workflows/_qc.yml"
+    with:
+      nixHash: ${{ needs.Checkout.outputs.nixHash }}
 
   BuildAndPackage:
     needs: [Checkout]
     uses: "./.github/workflows/_build_and_package.yml"
     with:
+      benchmark: true
```

```suggestion
      benchmark: false
```
Until I get that part fixed up